.github/workflows/release.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           # We can't verify all revisions due to the disk size limitation on GHA.
           ide-versions: |
             ideaIC:2024.1.6
-            ideaIC:2024.2.4
+            ideaIC:2024.3
```

Comment:
The new one should be added instead of replaced.

Reply:
Do you mean that should be 

ideaIC:2024.1.6
            ideaIC:2024.2.4
            ideaIC:2024.3

?

Reply:
Yeah.